cmd/utils/flags.go

```diff
@@ -72,1246 +72,1472 @@ var (
+		EnvVars: []string{"KLAYTN_NTP_DISABLE"},
 	}
 	NtpServerFlag = cli.StringFlag{
-		Name:   "ntp.server",
-		Usage:  "Remote ntp server:port to get the time",
-		Value:  "pool.ntp.org:123",
-		EnvVar: "KLAYTN_NTP_SERVER",
+		Name:    "ntp.server",
+		Usage:   "Remote ntp server:port to get the time",
+		Value:   "pool.ntp.org:123",
+		Aliases: []string{"common.ntp.server", "ns"},
+		EnvVars: []string{"KLAYTN_NTP_SERVER"},
 	}
 	NetworkTypeFlag = cli.StringFlag{
-		Name:   "networktype",
-		Usage:  "Klaytn network type (main-net (mn), service chain-net (scn))",
-		Value:  "mn",
-		EnvVar: "KLAYTN_NETWORKTYPE",
+		Name:    "networktype",
+		Usage:   "Klaytn network type (main-net (mn), service chain-net (scn))",
+		Value:   "mn",
+		Aliases: []string{},
+		EnvVars: []string{"KLAYTN_NETWORKTYPE"},
 	}
 	DbTypeFlag = cli.StringFlag{
-		Name:   "dbtype",
-		Usage:  `Blockchain storage database type ("LevelDB", "BadgerDB", "MemoryDB", "DynamoDBS3")`,
-		Value:  "LevelDB",
-		EnvVar: "KLAYTN_DBTYPE",
+		Name:    "dbtype",
+		Usage:   `Blockchain storage database type ("LevelDB", "BadgerDB", "MemoryDB", "DynamoDBS3")`,
+		Value:   "LevelDB",
+		Aliases: []string{"db.type", "migration.src.dbtype"},
+		EnvVars: []string{"KLAYTN_DBTYPE"},
 	}
 	SrvTypeFlag = cli.StringFlag{
-		Name:   "srvtype",
-		Usage:  `json rpc server type ("http", "fasthttp")`,
-		Value:  "fasthttp",
-		EnvVar: "KLAYTN_SRVTYPE",
-	}
-	DataDirFlag = DirectoryFlag{
-		Name:   "datadir",
-		Value:  DirectoryString{node.DefaultDataDir()},
-		Usage:  "Data directory for the databases and keystore. This value is only used in local DB.",
-		EnvVar: "KLAYTN_DATADIR",
-	}
-	KeyStoreDirFlag = DirectoryFlag{
-		Name:   "keystore",
-		Usage:  "Directory for the keystore (default = inside the datadir)",
-		EnvVar: "KLAYTN_KEYSTORE",
+		Name:    "srvtype",
+		Usage:   `json rpc server type ("http", "fasthttp")`,
+		Value:   "fasthttp",
+		Aliases: []string{"common.srvtype"},
+		EnvVars: []string{"KLAYTN_SRVTYPE"},
+	}
+	DataDirFlag = cli.PathFlag{
+		Name:    "datadir",
+		Value:   node.DefaultDataDir(),
+		Usage:   "Data directory for the databases and keystore. This value is only used in local DB.",
+		Aliases: []string{"common.datadir", "migration.src.datadir"},
+		EnvVars: []string{"KLAYTN_DATADIR"},
+	}
+	KeyStoreDirFlag = cli.PathFlag{
+		Name:    "keystore",
+		Usage:   "Directory for the keystore (default = inside the datadir)",
+		Aliases: []string{"common.keystore"},
+		EnvVars: []string{"KLAYTN_KEYSTORE"},
 	}
 	// TODO-Klaytn-Bootnode: redefine networkid
 	NetworkIdFlag = cli.Uint64Flag{
-		Name:   "networkid",
-		Usage:  "Network identifier (integer, 8217=Cypress (Mainnet) , 1000=Aspen, 1001=Baobab)",
-		Value:  cn.GetDefaultConfig().NetworkId,
-		EnvVar: "KLAYTN_NETWORKID",
+		Name:    "networkid",
+		Usage:   "Network identifier (integer, 8217=Cypress (Mainnet) , 1000=Aspen, 1001=Baobab)",
+		Value:   cn.GetDefaultConfig().NetworkId,
+		Aliases: []string{},
```

Comment:
Is it different from `p2p.network-id` ?

Reply:
Same thing. missed the alias. Thanks for your review in detail.